Cargo.lock

```diff
@@ -3454,20 +3455,21 @@ checksum = "1f3ccbac311fea05f86f61904b462b55fb3df8837a366dfc601a0161d0532f20"
 
 [[package]]
 name = "tokio"
-version = "1.40.0"
+version = "1.38.1"
```

Comment:
We shouldn't downgrade deps in this PR.

Reply:
Not 100% sure how this got downgraded, should be resolved now.